foreman/foreman.spec

```diff
@@ -625,7 +691,7 @@ Summary: Foreman sqlite support
 Group:  Applications/System
 Requires: %{?scl_prefix_ror}rubygem(sqlite3) >= 1.3.6
 Requires: %{?scl_prefix_ror}rubygem(sqlite3) < 1.4.0
-Requires: %{name} = %{version}-%{release}
+Requires: %{name}-core = %{version}-%{release}
```

Comment:
I've updated the plugins to not depend on the foreman-sqlite, to be independent of the foreman. All the other packages depend on foreman now